js/display-pictures.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { Description } from './mock.ts';
+
+const pictureList = document.querySelector('.pictures');
+const pictureTemplate = document.querySelector('#picture')?.querySelector('.picture');
+
+const picturesFragment = document.createDocumentFragment();
+
+const displayPictures = (dataArray: Description[]) => {
+	dataArray.forEach(({url, description, likes, comments}) => {
+		const pictureElement = pictureTemplate?.cloneNode(true) as HTMLAnchorElement;
+		const pictureTag = pictureElement.querySelector('.picture__img') as HTMLImageElement;
```

Comment:
Как альтернатива можно выходить из функции
```suggestion
		const pictureTag = pictureElement.querySelector<HTMLImageElement>('.picture__img');

		if (!pictureTag) {
			return;
		}
```